src/cli/Microsoft.DotNet.xunit.performance.runner.cli/project.json

```diff
@@ -20,17 +20,18 @@
   },
 
   "frameworks": {
-    "dnxcore50": {
-      "imports": "portable-net45+win8",
+    "netstandard1.5": {
```

Comment:
I copied it from here
https://github.com/dotnet/cli/blob/rel/1.0.0/TestAssets/TestProjects/PortableTests/PortableApp/project.json#L7
Will make the changes now